crates/services/gas_price_service/src/fuel_gas_price_updater.rs

```diff
@@ -172,6 +193,9 @@ impl From<AlgorithmUpdater> for UpdaterMetadata {
                 };
                 UpdaterMetadata::V0(metadata)
             }
+            AlgorithmUpdater::V1(_v1) => {
+                unimplemented!()
```

Comment:
Can we have a link to the issue where this is to be implemented for reference?

Reply:
addressed in a4e8a79ba